js/index.js

```diff
@@ -137,3 +138,115 @@ function creationOwlCarousel () {
             e.preventDefault();
     });
 };
+
+/**Coding section for Cards */
+if (document.readyState == 'loading') {
+    document.addEventListener('DOMContentLoaded', ready)
+} else {
+    ready();
+}
+
+function ready() {
```

Comment:
what this function name means? I can imagine it's something what should happen after page was loaded. Or maybe markers was rendered, or user finished shopping.